src/engine/native/node/native/src/index.js

```diff
@@ -137,7 +141,8 @@ class Native {
         noIPC.receive(taskID, args);
       };
 
-      return this._engine.init(options, noIPC);
+      this._engine.init(options, noIPC);
```

Comment:
Was the return value not necessary?

Reply:
It's not necessary, the init function from the core doesn't return anything